Demo/Demo/SceneDelegate.swift

```diff
@@ -1,15 +1,13 @@
 import UIKit
 
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
-
     var window: UIWindow?
 
-
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        guard let scene = (scene as? UIWindowScene) else { return }
```

Comment:
Won't this now lead to an XCode warning for an unused var?

Reply:
It doesn't look like I'm currently getting any issues in Xcode when building the project with this change 🤔

Reply:
It doesn't throw a swiftlint warning, but I do get an xcode warning on this line when i build the project

Reply:
😕 I wonder why I'm not getting a warning when I build the project

Reply:
Ah, nvm, needed to build the demo not paypal target 🤔

Reply:
Removing this line entirely for now to avoid xcode warnings or having to disable a linter rule since we don't need to reference this currently